docs/contribute.md

```diff
@@ -17,6 +17,7 @@ See the [ExecutableBooks developer guidelines](https://executablebooks.org/en/la
 - `src/` contains the code and assets that make up `thebe`. This is what you'll edit to make changes to the project.
 - `examples/` provides a few HTML examples of how `thebe` can be used. It is mostly for documentation
 
+(dev-install)=
```

Comment:
Not sure what this does.

Reply:
I see now below it is some way to make a reference. Never seen this markdown before.

Reply:
It is MyST!!